Return to the Bog

Alanii opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->

## About The Pull Request

Moves Low-Town into the old "Bog Town" position.
![image](https://github.com/user-attachments/assets/750108e6-ae3b-4cd3-8729-1c98ad927080)

Gives Vanguard an outpost within the "Bog Town" instead of a Bastion Gate.
This outpost has the only HERMES and SHYLOCK/NERVELOCK in Bog Town
![image](https://github.com/user-attachments/assets/d27ea875-8d32-4846-970a-e00bc30dc3ea)


Bastion Gate has been turned into a fortified castle gate.
(Not intended for toll shenanigans)
![image](https://github.com/user-attachments/assets/f9e47de8-ec38-4c34-8927-9fb65b7288da)

All SHYLOCK/NERVELOCK, HERMES, and SCOM have been removed from the "Bog Town" with the exception of the Vanguard Outpost.

The original Low Town area has been replaced with a mostly forested area.
![image](https://github.com/user-attachments/assets/77ee96f8-624d-451e-aed2-d758b4c00619)

Farm is now a proper farm again.
![image](https://github.com/user-attachments/assets/b5da0e30-5252-4938-a6fa-e3f8b35899d2)


## Why It's Good For The Game

This is pulled from recent survey results showcasing the following interest in majority fashion, along with Useroth's wishes to return to the bog in some capacity.

- Majority wished Low Town returned to the River
- Majority wished for Vanguard to stop being relegated to "Papers Please"

The goal of this PR is to bring back some "whatever the word is for reminiscing about the past" to Ratwood. This manner is done without compromising the idea of making the "Bog Town" compromised in security. If people who reside in this area need refuge, they can seek the Vanguard Outpost, which was been designed as a fortified position within the "Bog Town."

Arguably, this map is just a proper "temporary" fall in until a more competent mapper can go and make this a lot better.

## Proof of Testing (Required)
![image](https://github.com/user-attachments/assets/9aa0cdd1-fa04-470f-bef5-21ad3735d6ef)

Radioaction-Cyber commented:
Very good change. Beautiful map. 10/10.
This'll make Vanguard more of a protection role, rather than a toll post. People can actually play the role as it was intended. 

I do think there may be a benefit to adding more houses right outside the Bastion gate-- where Low Town used to be. Not too many, but some scattered through could be good for towners to use.

Hmeister-real commented:
keeping the little old-bastion gate cell is un-needed, maybe make the pens in the farm a little smaller aswell, they look too large, but besides that this looks great!

SnipstheFox commented:
Where is the mine?

Alanii commented:
> Where is the mine?

North of the town square. Gave it some skeletons for vanguard to deal with and increased the yield some.

Tastyfish commented:
> Very good change. Beautiful map. 10/10. This'll make Vanguard more of a protection role, rather than a toll post. People can actually play the role as it was intended.
> 
> I do think there may be a benefit to adding more houses right outside the Bastion gate-- where Low Town used to be. Not too many, but some scattered through could be good for towners to use.

I agree, it seems bizarre that there's just a forest in the middle of town. Not sure what a good replacement is though. I suppose people can use it to build.

Alanii commented:
Updated changelog for everything

- Rearranged Low town
  - Moved Vanguard outpost and made it centerpiece of town
  - Rearranged Low town entirely
- Moved Stables (again)
- Gave bandits an alternate fixed exit point
- Gave a specific area in low-town chapel for priest ability usage
- Rerarranged Lighting
